tap_typeform/__init__.py

```diff
@@ -81,11 +84,25 @@ def sync(atx):
     LOGGER.info('--------------------')
 
 
+def validate_form_ids(config):
```

Comment:
We should have tests for this function to confirm that its working like we expect with the call to `difference`